docs/user-guide.md

```diff
@@ -498,6 +539,46 @@ You can export and import prompts.
 
 ![](./img/import-export-prompt.png)
 
+### Publish
+
+You can publish prompts to make them available for the target audience. You can also unpublish them to withdraw from the public use.
+
+**Note**: to publish a shared prompt, [duplicate](#duplicate-1) it and then publish. 
+
+**To publish a prompt**: 
+
+1. Click **Publish** in the prompt's menu. 
+
+![](./img/publish-prompt.png)
+
+2. In the pop-up window:
+   * Enter **publication request name** in the upper area.
+   * In **Publish to**, select the organization or a folder to publish your prompt into.
+   * In **Allow access...**, specify access rules. For example Role-Equals-Admin. **Note**: the available roles are defined in each organization individually. Contact your support to find out the rules are applicable in your organization.
+   * In **Prompts**, if you have selected a folder, you can choose what prompts you want to publish.
+   * Click **Send request** to send your publication request to the administrator. **Note**: you can revoke your publication request only prior it has been approved.
```

Comment:
In fact, right now the UI doesn't allow you to do a revoke before approval.

Reply:
fixed